client/src/components/RelatedItemsAndComparisons/Card.jsx

```diff
@@ -1,14 +1,22 @@
 /* eslint-disable jsx-a11y/no-static-element-interactions */
 /* eslint-disable jsx-a11y/click-events-have-key-events */
-// This child component will contain a single product card
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
 // TODO: have a set dimension size for each card
 // TODO: import image from API request
 
 function Card({
-  name, category, price, tempImg,
+  product, name, category, price,
 }) {
+  // eslint-disable-next-line no-unused-vars
+  const [productStyles, setProductStyles] = useState({});
+  const [salePrice, setSalePrice] = useState(null);
+  const [defaultImages, setDefaultImages] = useState([]);
```

Comment:
These images are coming from the API so they arent the default images right?

Reply:
Yes but each product has a style object that has a 'default?' property = true which dictates that this style and images are the default for this product